support/cas-server-support-otp-mfa-core/src/main/java/org/apereo/cas/otp/validator/OneTimeTokenAccountValidator.java

```diff
@@ -0,0 +1,47 @@
+package org.apereo.cas.otp.validator;
+
+import org.apereo.cas.authentication.Credential;
+import org.apereo.cas.authentication.OneTimeTokenAccount;
+import org.apereo.cas.authentication.credential.OneTimeTokenCredential;
+
+import lombok.val;
+
+/**
+ * This is {@link OneTimeTokenAccountValidator}.
+ *
+ * Provides a common interface for objects that can
+ * validate OTP credentials.
+ *
+ * @author Hayden Sartoris
+ * @since 6.2.0-RC5
+ */
+public interface OneTimeTokenAccountValidator {
+
+    /**
+     * Is the token valid for the given account?
+     *
+     * Assumes parsing logic has taken place.
```

Comment:
This statement is unclear. What is "parsing logic"?

Reply:
Admittedly doesn't clarify much if the parsing is String to int, and this only accepts an int anyway.